coreneuron/utils/profile/cuda_profile.cu

```diff
@@ -23,7 +23,7 @@ void print_gpu_memory_usage() {
     double free_db = (double) free_byte;
     double total_db = (double) total_byte;
     double used_db = total_db - free_db;
-    printf("\n  => GPU MEMORY USAGE (MB) : Used = %f, Free = %f MB, Total = %f",
+    printf(" GPU Memory Usage (MB) : Used = %f, Free = %f MB, Total = %f\n",
            used_db / 1024.0 / 1024.0,
```

Comment:
Nitpick - what if we do little refactoring and print cpu + gpu memory report in same message?

Reply:
You mean print this on the same line? I am afraid that it would print a very long string which wouldn't look nice in small terminals. If you think that's okay I can make it like that though